app/src/main/java/com/iron/espresso/presentation/home/setting/SettingFragment.kt

```diff
@@ -20,6 +23,32 @@ class SettingFragment : Fragment() {
     ): View? {
         binding = DataBindingUtil.inflate(inflater, R.layout.fragment_setting, container, false)
         binding.lifecycleOwner = this
+
+        val settingCategoryList = arrayListOf<CategoryItem>()
+        for ((count, i) in resources.getStringArray(R.array.setting_category).withIndex()) {
+            val arrayList = arrayListOf<ItemList>()
+            when (count) {
+                0 -> for (j in resources.getStringArray(R.array.category_account)) {
+                    arrayList.add(ItemList(j))
+                }
```

Comment:
이런거 잘 찾아서 적용해보세요. 지금 전체적으로 리스트 처리의 복잡도가 높은 상태로 보여요
https://0391kjy.tistory.com/38
```suggestion
                0 -> arrayList.addAll(resources.getStringArray(R.array.category_account).map { ItemList(it) })               
```